studio-docs/source/contracts.mdx

```diff
@@ -91,27 +91,30 @@ type Query {
   topProducts: [Product!]!
 }
 
-type Product @key(fields: "upc") {
+# All fields on the "Product" object inherit the "partner" tag so we can avoid tagging them explicitly
```

Comment:
All fields of the Product object type automatically inherit...

explicitly -> individually (either works fine if you'd rather keep 😄)